src/components/layout/organize/TabbedLayout.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import TabbedLayout from './TabbedLayout';
+import { fireEvent, render } from 'test-utils';
+
+const mockTabbedLayoutProps = {
```

Comment:
Maybe we should expose the feature you implemented as a prop which can be passed to the layout component? I wonder if we want it by default on every page? @richardolsson what do you think?

Reply:
Yes I was thinking about a 'collapsable' prop... Will leave for now

Somewhat related musing: had a thought about a container component that can allow any child component to be expanded to fullscreen, something like <Fullscreenable><Big.Table.Component /></Fullscreenable>, allowing any view a user might conceivably want to make big, to be made big...
I.e. the focus would be on expanding the view to cover other things, rather than moving the other things out of the way.